code/obj/item/organs/pancreas.dm

```diff
@@ -39,3 +41,18 @@
 	robotic = 1
 	edible = 0
 	mats = 6
+
+	on_life(var/mult = 1)
+		if (!..())
+			return 0
+		if(!donor.reagents)
+			return 1
+		if((donor.reagents.get_reagent_amount("sugar") < 20 || emagged) && donor.reagents.get_reagent_amount("glaucogen") <= 5)
```

Comment:
I'm not really sure what the purpose of this reagent, glaucogen is. At its declaration, it doesn't have any particular interactions on mob life and it doesn't decay into anything. 

It seems like the cyber pancreas just gives you up to ~5u of glaucogen while you have no or less than 20 sugar in your system or if it's emagged. Seems like it's basically just a counter, but I can't tell what it's used for because nothing checks it other than here.

Reply:
Glaucogen is a child of corn syrup - decays into 1.2(*mult) units of sugar on mob life